CHANGELOG.md

```
@@ -307,6 +307,44 @@ Release 1.6.0.
 
 ---
 
+## [1.7.0] - 2026-08-10
+
+Release 1.7.0.
+
+## Changes
```

markdownlint-cli2 (0.23.2)

[warning] Multiple headings with the same content

(MD024, no-duplicate-heading)